v03_pipeline/migrations/annotations/0002_add_remap_pedigree_hash.py

```diff
@@ -0,0 +1,28 @@
+import hail as hl
+
+from v03_pipeline.lib.migration.base_migration import BaseMigration
+from v03_pipeline.lib.model import DatasetType, ReferenceGenome
+
+
+class AddRemapPedigreeHash(BaseMigration):
+    @property
+    def reference_genome_dataset_types() -> (
+        frozenset[tuple[ReferenceGenome, DatasetType]]
+    ):
+        return frozenset(
+            (
+                (ReferenceGenome.GRCh37, DatasetType.SNV_INDEL),
+                (ReferenceGenome.GRCh38, DatasetType.SNV_INDEL),
+                (ReferenceGenome.GRCh38, DatasetType.MITO),
+                (ReferenceGenome.GRCh38, DatasetType.GCNV),
+                (ReferenceGenome.GRCh38, DatasetType.SV),
+            ),
+        )
+
+    @staticmethod
+    def migrate(ht: hl.Table) -> hl.Table:
+        return ht.annotate_globals(
```

Comment:
Do you need to add this migration to the `migrations` globals? 
I don't see it in the BaseMigration but that behavior could belong there.

Reply:
It's in [BaseMigrateTask](https://github.com/broadinstitute/seqr-loading-pipelines/blob/dev/v03_pipeline/lib/tasks/base/base_migrate.py#L47).  There's a distinction between the `BaseMigration` (which is an interface that the migration definitions implement) and the `BaseMigrateTask`, which is something that a "table" (lookup/annotations or in theory projects) inherit from.